pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -500,6 +508,44 @@ func (oe *operationExecutor) generateDetachVolumeFunc(
 	}
 
 	return func() error {
+		if verifySafeToDetach {
+			// Fetch current node object
```

Comment:
I mean when this function is invoked with verifySafeToDetach=false, and the thing is in fact reported as "in use" by a node (so get the node and check), but we're going to go ahead and detach anyway because of the false boolean